sample/app/src/main/res/values/mparticle.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- ex. src/main/res/values/mparticle.xml -->
+<resources>
+    <bool name="mp_enableDebugMode">true</bool>
+    <bool name="mp_enablePush">true</bool>
+    <string name="mp_pushSenderId">433406026043</string>
+    <string name="mp_key">03b7f667f2fbd2439f97b46fd3604c93</string>
```

Comment:
please remove these from sample app. replace them with something like `<REPLACE ME>`

Reply:
will update PR to replace 'mp_pushSenderId' and 'mp_key' values with <REPLACE ME>